src/main.c

```diff
@@ -74,41 +73,9 @@ int main(int argc, char *argv[])
         goto create_player_fail;
     }
 
-    const rect_t platforms_rects[] = {
-        { .x = 0.0f,
-          .y = GROUND_LEVEL + 50.0f,
-          .w = 50.0f,
-          .h = 50.0f },
-        { .x = 300.0f,
-          .y = GROUND_LEVEL,
-          .w = 50.0f,
-          .h = 50.0f },
-        { .x = 150.0f,
-          .y = GROUND_LEVEL + 50.0f,
-          .w = SCREEN_WIDTH - 150.0f,
-          .h = 50.0f },
-        { .x = 0.0f,
-          .y = GROUND_LEVEL + 100.0f,
-          .w = 50.0f,
-          .h = 50.0f },
-        { .x = 150.0f,
-          .y = GROUND_LEVEL + 100.0f,
-          .w = 50.0f,
-          .h = 50.0f },
-        { .x = 0.0f,
-          .y = GROUND_LEVEL + 150.0f,
-          .w = 50.0f,
-          .h = 50.0f },
-        { .x = 150.0f,
-          .y = GROUND_LEVEL + 150.0f,
-          .w = 50.0f,
-          .h = 50.0f }
-    };
-    platforms_t *platforms = create_platforms(
-        platforms_rects,
-        sizeof(platforms_rects) / sizeof(rect_t));
+    platforms_t *platforms = load_platforms_from_file("./platforms.txt");
```

Comment:
Maybe have a default hardcoded set of rects when the file not found?

Reply:
Nah, to much of an effort. It also requires adding a check on reloading and stuff. Not worth it.